App.jsx

```diff
@@ -0,0 +1,15 @@
+const App = () => {
+	return (
+	  <div>
+	    <h1>Hello world</h1>
+	    <UserCard userId="1" />
```

Comment:
This is just a small thing, but React will actually let you render out an array of elements. This means that you can use the `map` function to build out a group of components like this. You could imagine doing something like:

```jsx
const userIds = [1, 2, 3, 4, 5, 6];

return (
  <div>{userIds.map((userId) => <UserCard userID={userId} /> )}</div>
);
```

The nice thing about that is, if you want to render additional cards, you can simply update the array with new values. You could even imagine pulling that list of IDs from an API or something later at some point, too.

Reply:
Oh wow, I did not know that will refactor to this, thank you !